app/uk/gov/hmrc/cipphonenumberverification/audit/AuditEvent.scala

```diff
@@ -14,12 +14,7 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.cipphonenumberverification.models
+package uk.gov.hmrc.cipphonenumberverification.audit
 
-import play.api.libs.json.{Json, Reads}
+abstract class AuditEvent(phoneNumber: String, passcode: String)
 
-case class GovUkNotificationStatus(status: String)
```

Comment:
is this class no longer needed?

Reply:
theres now a class called GovUkNotificationStatusResponse for the deserialising of the response from GovNotify, as I need more than the status now.
I think its just the way IntelliJ does the refactoring, it shows up as being removed.

`case class GovUkNotificationStatusResponse(phone_number: String, body: String, status: String)`